packages/dart_frog_cli/lib/src/daemon/domain/dev_server_domain.dart

```diff
@@ -43,6 +43,12 @@ class DevServerDomain extends DomainBase {
       );
     }
 
+    final host = request.params?['host'];
+
+    if (!_isValidHost(host)) {
```

Comment:
This makes a host parameter required. As far as I am concerned this is a breaking change in the Dart Frog daemon because it will break all daemon users currently using the dev server domain (for example the VS Code extension).

We should either consider making it optional or bumping the daemon version? (cc: @renancaraujo)

Reply:
@alestiago isn't the host always required? Even if the use don't inform one, we need to assume it is localhost?

If the daemon don't already have a default for this, I would say that we should follow the path of assuming localhost as default instead of introducing a breaking change.

Reply:
@alestiago, I think @erickzanardo is correct. We can add localhost if the host is null. 

`final host = request.params?['host'] ?? 'localhost';`

Reply:
> @alestiago isn't the host always required?

I believe, as of today, the _dev_server.start_ method does not require the host; see the [docs](https://dartfrog.vgv.dev/docs/advanced/daemon#method-start).

>  We can add localhost if the host is null.

That will make it a non-breaking change, I'm happy with both approaches.

Reply:
@alestiago I suggest that let's make it a non-breaking change. You can create a new issue for Deamon to introduce breaking change.